data/school-types.csv

```diff
@@ -0,0 +1,72 @@
+establishment_governance,Establishment governance,Independent,establishment_type_group,,IN
+establishment_governance,Establishment governance,Non-maintained,establishment_type_group,,NM
+establishment_governance,Establishment governance,City technology college,establishment_type_group,,CT
+establishment_governance,Establishment governance,Academy,establishment_type_group,,CA
+education_phase,Education phase,All-through,establishment_type_group,,AT
+education_phase,Education phase,Alternative provision,establishment_type_group,,PR
+education_phase,Education phase,Early years settings,establishment_type_group,,
+education_phase,Education phase,Independent school,establishment_type_group,,
+education_phase,Education phase,Middle (deemed primary),establishment_type_group,,MP
+education_phase,Education phase,Middle (deemed secondary),establishment_type_group,,MS
+education_phase,Education phase,No establishment,establishment_type_group,,
+education_phase,Education phase,Nursery,establishment_type_group,,NS
+education_phase,Education phase,Primary,establishment_type_group,,PS
+education_phase,Education phase,Secondary,establishment_type_group,,SS
+education_phase,Education phase,Special,establishment_type_group,,SP
+education_phase,Education phase,Further education,establishment_type_group,,
+education_phase,Education phase,Higher education,establishment_type_group,,
+education_phase,Education phase,Total,establishment_type_group,,
+establishment_type,Establishment type,Academy 16 to 19 converter,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Academy 16 to 19 sponsor led,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Academy converter,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Academy sponsor led,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,All establishments,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,City technology college,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Community school,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Foundation school,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Free school,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Free school 16 to 19,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Hospital special school,education_phase;establishment_type_group;establishment_governance,,
+establishment_type,Establishment type,Other FE sector college,education_phase;establishment_type_group;establishment_governance,,
```

Comment:
What do the multiple options here mean? Is that saying there's there's different available options, or that at some point we'd expect to have all of these as layers in the same hierarchy?

Reply:
Those are to denote multiple possible choices for grouping column (given the current 2-tier hierarchy) that teams might have in their data.